specification/translation/Azure.AI.DocumentTranslation/models.tsp

```diff
@@ -154,7 +154,7 @@ model SourceInput {
   language?: string;
 
   @doc("Storage Source")
-  storageSource?: StorageSource;
+  translationStorageSource?: TranslationStorageSource;
```

Comment:
This is changing an actual property that the service sends back...these renames should only be done in client.tsp. Same comment for all other property updates. Model name renames are ok since the model name is not sent over the wire.

Reply:
Updated